diff --git a/styles/PerkSelectModal.styles.js b/styles/PerkSelectModal.styles.js
index 2004309..445aaab 100644
--- a/styles/PerkSelectModal.styles.js
+++ b/styles/PerkSelectModal.styles.js
@@ -49,7 +49,7 @@ const styles = StyleSheet.create({
     paddingHorizontal: 10,
     paddingBottom: 8,
     borderTopWidth: 1,
-    borderTopColor: '#cbd5e1',
+    borderTopColor: 'rgb(221, 221, 221)',
   },
   searchInput: {
     borderWidth: 1,
